src/main/java/bio/terra/service/dataset/flight/ingest/IngestUtils.java

```diff
@@ -200,14 +201,13 @@ public static PdaoLoadStatistics getIngestStatistics(FlightContext context) {
       };
 
   public static Stream<JsonNode> getJsonNodesStreamFromFile(
-      GcsPdao gcsPdao,
+      CloudFileReader cloudFileReader,
       ObjectMapper objectMapper,
       IngestRequestModel ingestRequest,
-      Dataset dataset,
+      String cloudEncapsulationId,
```

Comment:
`cloudEncapsulationId` is a pretty clunky name, but I'm struggling to come up with a term that encompasses both a GCP `projectId` and Azure `tenantId`

Reply:
Yeah that's tricky...how about `cloudLocationId`?